src/scenes/home/profile/profile.js

```diff
@@ -16,13 +16,13 @@ const Profile = ({ verified }) => (
         <IconCard
```

Comment:
let's do `{verified && <IconCard ... />}` instead of conditionally having a URL for both